client/my-sites/email/inbox/mailbox-selection-list/progress-line-style.scss

```
@@ -0,0 +1,101 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+.progress-line__container {
+	position: fixed;
+	top: 50%;
+	left: 50%;
+	transform: translate( -50%, -50% );
+
+	@include break-large {
+		transform: none;
+	}
+
+
+	.progress-line__status-text-container {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		margin-bottom: 10px;
+
+		@include break-large {
+			margin-bottom: 20px;
+		}
+
+		.progress-line__status-text {
+			font-family: $sans;
+			font-weight: 400;
+			font-size: $font-title-small;
+			line-height: 22px;
+
+			@include break-large {
+				font-size: $font-title-medium;
+				line-height: 28px;
+			}
+		}
+	}
+
+	.progress-line__loading-container {
+		position: relative;
+		width: 280px;
+		margin: 0 auto;
+		border-radius: 2px;
+		box-sizing: border-box;
+
+		@include break-medium {
+			width: 335px;
+		}
+
+		@include break-large {
+			width: 400px;
+		}
```

I think we can also use flexbox for this element, as well as possibly making it more responsive using a baseline percentage-based `width` along with a `max-width` at various breakpoints.
```suggestion
		align-self: center; // assuming parent has `display: flex;`; this may not be necessary
		border-radius: 2px;
		box-sizing: border-box;
		max-width: 280px;
		width: 50%;

		@include break-medium {
			max-width: 335px;
		}

		@include break-large {
			max-width: 400px;
		}
```